server/scanner/LibraryScanner.js

```diff
@@ -647,14 +650,15 @@ class LibraryScanner {
       if (newLibraryItem) {
         const oldNewLibraryItem = Database.libraryItemModel.getOldLibraryItem(newLibraryItem)
         SocketAuthority.emitter('item_added', oldNewLibraryItem.toJSONExpanded())
+        this.notificationManager.onItemsAdded([oldNewLibraryItem])
       }
       itemGroupingResults[itemDir] = newLibraryItem ? ScanResult.ADDED : ScanResult.NOTHING
     }
 
     return itemGroupingResults
   }
 }
-module.exports = new LibraryScanner()
+module.exports = new LibraryScanner(new NotificationManager())
```

Comment:
The server doesn't run because `NotificationManager` is not defined.
We wouldn't want to instantiate a new instance of NotificationManager though because it is already instantiated in Server.js.
The NotificationManager could be made a singleton like the LibraryScanner so it can be imported anywhere

Reply:
I updated it to a singleton so you can merge master to fix
https://github.com/advplyr/audiobookshelf/commit/7cd8d7f44d999565e6ca2941704158519c78b869

Reply:
Interesting, I ran the server just fine with the new instantiation of `NotificationManager` here. Maybe I missed something when migrating my changes from a branch off master to my fork. Agreed, a singleton would have been better, but I didn't feel comfortable making a sweeping change like that, so I appreciate the enhancement. I will update accordingly and resubmit the PR.